.github/workflows/ci.yaml

```
@@ -319,6 +327,16 @@ jobs:
           -----END PGP PUBLIC KEY BLOCK-----
           EOF
 
+      - name: Login to the registry if needed
+        run: |
+          if [ "$TARGET" = "ibs-released" ]; then
+            echo $REGISTRY_LOGIN_PASSWORD | podman login -u $REGISTRY_LOGIN_USERNAME --password-stdin registry.suse.com
+          fi
+        env:
+          REGISTRY_LOGIN_USERNAME: ${{ secrets.REGISTRY_LOGIN_USERNAME }}
+          REGISTRY_LOGIN_PASSWORD: ${{ secrets.REGISTRY_LOGIN_PASSWORD }}
+          TARGET: ${{ matrix.testing_target != '' && matrix.testing_target || 'obs' }}
```

A bit simpler imho, but I don't insist on this change

```suggestion
      - name: Login to the registry if needed
        run: |
          echo $REGISTRY_LOGIN_PASSWORD | podman login -u $REGISTRY_LOGIN_USERNAME --password-stdin registry.suse.com
        if: ${{ matrix.testing_target == 'ibs_released' }}
        env:
          REGISTRY_LOGIN_USERNAME: ${{ secrets.REGISTRY_LOGIN_USERNAME }}
          REGISTRY_LOGIN_PASSWORD: ${{ secrets.REGISTRY_LOGIN_PASSWORD }}
```